golos.vesting/golos.vesting.hpp

```diff
@@ -34,10 +34,25 @@ class vesting : public contract {
         const auto& balance = balances.get(sym.raw());
         return balance.vesting;
     };
-    inline asset get_account_effective_vesting(name account, symbol_code sym) const;
-    inline asset get_account_available_vesting(name account, symbol_code sym) const;
-    inline asset get_account_unlocked_vesting(name account, symbol_code sym) const;
-    inline bool balance_exist(name owner, symbol_code sym) const;
+    static inline asset get_account_effective_vesting(name code, name account, symbol_code sym) {
```

Comment:
See changes and comment https://github.com/GolosChain/golos-smart/pull/325#discussion_r238925364